### PR TITLE
Aws doc corrections

### PR DESCRIPTION
**Pull request by reshmabidikar**
Includes the following:

1.  Corrections to the singletier/multitier/cf maintenance docs as required
2. Moved common content to separate files and included this via `includes`

I've filed https://github.com/killbill/killbill-docs/issues/562 to track the work that is left

### CODE REVIEW

**Review by pierre — APPROVED** [review]
Very nice! 👏 

Could you just fix the warnings (I believe Markdown format was used, while the files are Asciidoc)? Then, good to merge!

**Comment by sbrossie** [comment]
> Very nice! 👏
> 
> Could you just fix the warnings (I believe Markdown format was used, while the files are Asciidoc)? Then, good to merge!

Agreed. @reshmabidikar  Also I would suggest to take a look at the doc once regenerated after the merge since there is a lot of 'style' involved, and this is hard to really review with a PR.

File: userguide/aws/how-to-maintain-a-cloud-formation-system.adoc [hunk 1]
```diff
@@ -89,39 +105,14 @@ If you update any configuration file, you will need to restart the service. To d
...
-> grep 'org.killbill.dao' /var/lib/killbill/config/killbill.properties
-org.killbill.dao.password=killbill
-org.killbill.dao.url=jdbc:mysql:aurora://mystack-test-rdscluster-1qwiqitatcb9m.cluster-cah16olm8gkg.us-east-1.rds.amazonaws.com:3306/killbill
-org.killbill.dao.user=killbill
-----
-
-Based on this info, the following command would allow you to get a `mysql` prompt:
-
-[source,bash]
+> sudo service kaui restart
```
Comment (vnandwana) [inline userguide/aws/how-to-maintain-a-cloud-formation-system.adoc]: Does this work? AFAIK, we're not starting Kaui as a service.

File: userguide/aws/includes/healthcheck.adoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
+The healthcheck report checks the health of various software components, and determines if any queues are growing improperly over time. To create this report, issue the following command:
+
+`curl \http://127.0.0.1:8080/1.0/healthcheck`
```
Comment (vnandwana) [inline userguide/aws/includes/healthcheck.adoc:3]: please remove the backslash before the URL

File: userguide/aws/includes/logs.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Tomcat/KB/Kaui logs are under `/var/lib/tomcat/logs/`:
+
+* Kaui log: `/var/lib/tomcat/logs/kaui.out`
+* Kill Bill server log: `/var/lib/tomcat/logs/killbill.out`
```
Comment (vnandwana) [inline userguide/aws/includes/logs.adoc:4]: The short description that was there earlier was good. We can add a line at the top stating that, by default, the log level is configured as `INFO` for all logs.

File: userguide/aws/includes/system-information.adoc [hunk 4]
```diff
@@ -0,0 +1,50 @@
+For a detailed system information report, showing the current version of each component, use the following command:
+
+`curl -u <ADMIN>:<INSTANCE_ID> \http://127.0.0.1:8080/1.0/kb/nodesInfo`
```
Comment (vnandwana) [inline userguide/aws/includes/system-information.adoc:3]: is the backslash before the URL required?

File: userguide/aws/how-to-maintain-a-cloud-formation-system.adoc (hunk 1 above)
Comment (reshmabidikar) [reply userguide/aws/how-to-maintain-a-cloud-formation-system.adoc]: Fixed

File: userguide/aws/includes/logs.adoc (hunk 3 above)
Comment (reshmabidikar) [reply userguide/aws/includes/logs.adoc:4]: > The short description that was there earlier was good.

Not sure what this refers to but I've made some changes. Please take a look and if this is not what you mean, could you please provide further details?

> We can add a line at the top stating that, by default, the log level is configured as INFO for all logs.

Added this line.

File: userguide/aws/includes/healthcheck.adoc (hunk 2 above)
Comment (reshmabidikar) [reply userguide/aws/includes/healthcheck.adoc:3]: The backslash is intentional, local URLs like `http://localhost:8080/` need to be escaped with a backslash for SEO purpose.

**Review by vnandwana — APPROVED** [review]

**Pull request merged**